backend/src/Designer/Models/Authorization/PolicyRule.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace Altinn.Studio.Designer.Models.Authorization
+{
+    public class PolicyRule
```

Comment:
A bit off-topic, but did we ever figure out how to handle isDelegable=false? In the JSON variant this probably belongs as a boolean property in `PolicyRule`.

Reply:
Nope. We have not solved that yet